bugsnag-android-core/src/main/java/com/bugsnag/android/ManifestConfigLoader.kt

```diff
@@ -107,7 +107,7 @@ internal class ManifestConfigLoader {
             if (data.containsKey(ENABLED_RELEASE_STAGES)) {
                 enabledReleaseStages = getStrArray(data, ENABLED_RELEASE_STAGES, emptySet())
             }
-            ignoreClasses = getStrArray(data, IGNORE_CLASSES, ignoreClasses)
+            discardClasses = getStrArray(data, IGNORE_CLASSES, discardClasses)
```

Comment:
Need to rename the manifest key name - IGNORE_CLASSES.

Reply:
Renamed the manifest key.